### PR TITLE
Release test plan: use the Avocado COPR repo 

### PR DESCRIPTION
**Pull request by clebergnu**
Instead of, now, re-building the packages locally.  The COPR repo already builds them for different distros and on two architectures.

### CODE REVIEW

File: examples/testplans/release.json [hunk 1]
```diff
@@ -9,12 +9,6 @@
 	{"name": "Avocado source does not contain spelling errors",
 	 "description": "On your development machine, on a freshen Avocado source to be released, run `$ make spell`. Expected result: Make command should say OK."},
 
-	{"name": "Avocado RPM build",
-	 "description": "On your development machine, build the Avocado RPM packages using: `$ make rpm`. Expected result: SRPM and RPM files at `BUILD/RPM/python-avocado-x.y.z-r.distro.{src,noarch}.rpm`"},
-
-	{"name": "Avocado RPM lint",
-	 "description": "With the built RPMs, check if there are no unexpected packaging errors or warnings by running: `$ rpmlint BUILD/RPM/python*-avocado-x.y*.rpm`.  Expected result: 3 packages and 0 specfiles checked; 0 errors, 0 warnings. (some missing man-pages can be ignored)"},
-
```
Comment (ldoktor) [inline examples/testplans/release.json]: Without these steps the next step doesn't make sense. Would you please merge the commits as they need to be atomic?

File: examples/testplans/release.json [hunk 2]
```diff
@@ -9,12 +9,6 @@
 	{"name": "Avocado source does not contain spelling errors",
 	 "description": "On your development machine, on a freshen Avocado source to be released, run `$ make spell`. Expected result: Make command should say OK."},
 
-	{"name": "Avocado RPM build",
-	 "description": "On your development machine, build the Avocado RPM packages using: `$ make rpm`. Expected result: SRPM and RPM files at `BUILD/RPM/python-avocado-x.y.z-r.distro.{src,noarch}.rpm`"},
-
-	{"name": "Avocado RPM lint",
```
Comment (ldoktor) [inline examples/testplans/release.json]: I haven't found any checks on copr with regards to warnings, will it fail? If not than we still need to perform this step on the downloaded package...

File: examples/testplans/release.json (hunk 1 above)
Comment (clebergnu) [reply examples/testplans/release.json]: Sure!

File: examples/testplans/release.json (hunk 2 above)
Comment (clebergnu) [reply examples/testplans/release.json]: I'm adding the `rpmlint` step to the machine where the packages are installed to avoid the duplicate download.

**Pull request closed without merging**